assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -91,6 +91,17 @@ const NavArea = styled(Area)`
 
 const MenuArea = styled(Area).attrs({ area: 'menu' })``;
 
+const EditorVersion = styled.div`
+  display: inline-block;
+  position: relative;
+  margin-left: 14px;
+  bottom: 30px;
+  font-size: 12px;
+  line-height: 20px;
+  letter-spacing: 0.0133em;
```

Comment:
I wasn't sure if I should hard code these here, use an existing option from our theme, or add to our theme.  Thoughts?

Reply:
Yea I'd like clarity on this too. Feels kinda gross to piecemeal different attributes of different fonts in the theme object together to match the designs.

Reply:
+1 to add to the theme

Reply:
Not sure how much we wanna alter the theme at this moment, but might be nice to just create generic rhythms for these properties in the theme and then create helpers for functionality based naming:
```
// theme/theme.js
export const theme = {
  fontSize: {
    mobile: {
      100: '12px',
      200: '16px',
    },
    desktop: {
      100: '14px',
      200: '18px',
      ...
    }
  },
};

// theme/tab.js
export const tab = ({ theme }) => css`
  font-size: ${theme.fontSize.mobile[100]};
  font-family: ${theme.fontFamily.secondary};
  ...
  @media ${theme.breakpoint.desktop} {
    font-size: ${theme.fontSize.desktop[100]};
    ...
  }
`;

// someComponent.js
import { tab } from '../../theme';

export const SomeComponent = styled.div`
  ${tab}
  // additional styles
`;
```

I think @BrittanyIRL did something similar to this in the dashboard. Would be nice to unify